optaweb-employee-rostering-backend/pom.xml

```diff
@@ -230,65 +235,53 @@
             </excludedTestClasses>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>net.revelc.code.formatter</groupId>
+          <artifactId>formatter-maven-plugin</artifactId>
+          <version>2.11.0</version>
+          <configuration>
+            <configFile>${format.directory}/eclipse-format.xml</configFile>
```

Comment:
1. The plugin configuration should go to the parent's `<pluginManagement>` so that it can be reused in child modules.
2. The plugin should be added to the build (`<build>/<plugins>`) in modules that contain Java source code (backend, benchmark).